src/bootstrap.jl

```diff
@@ -444,7 +448,7 @@ Return the shortest interval containing `level` proportion for each parameter fr
     a breaking change.
 """
 function shortestcovint(bsamp::MixedModelFitCollection{T}, level=0.95) where {T}
-    allpars = bsamp.allpars
+    allpars = bsamp.allpars  # probably simpler to use .tbl instead of .allpars
```

Comment:
Sure.  The result will need to be expressed slightly differently because the column names of the table are named like :β1,  :β2,  :β3, :β4 and the current output uses the three-column naming convention.